src/module.js

```diff
@@ -100,27 +98,31 @@ function processModule( name, options, executeNow, modifiers = {} ) {
 	}
 }
 
+let focused = false;
+
 export default function module( name, options, executeNow ) {
 	if ( focused && !isParentModuleInQueue() ) {
+		config.currentModule.closed = true;
```

Comment:
It took me a while to realize that this is actually mutating the "previous" module. The one we are processing now, never makes it into the registry. This feels a bit fragile as it means `test()` calls are thus interacting with a closed module and I can see that leading to some confusing failures in the future.

On the other hand, given the way that `only` is meant to act like a `filter`, the module isn't meant to be visible in any way. Perhaps, though, it would be best to still let it enter the registry, but as an empty module that (if all goes well) will never start or stop, because it is empty. Afaik that is how empty modules are treated.

If empty modules are exposed to reporters or callbacks in some way, then we'd need some additional logic to make sure e.g. empty modules with `currentModule.ignored` get bypassed in our events/callback layer.

Reply:
Let me know if you like that direction. I haven't thought it through fully, so it's not a strong suggestion, just an idea.

For now my only suggestion would be to add an inline comment indicating what we have done, which is to keep the previous module on the stack and that "closed" means it was implicitly closed by a subsequent `module()` call.

Reply:
I'm certainly intrigued by that suggestion, and the notion of `currentModule.ignored` has a much more natural feel and intuition.

I think it's inside `module.only` where we set `config.modules.length = 0;` to effectively wipe out the others that were registered before. I'm trying to make that work with something like
```
config.modules.forEach( module => {
   module.ignored = true;
} );
```
but I'm definitely wrestling with the unittests - I don't have everything quite right.

I'll keep this open and continue noodling. I agree we should document the exposed property, and `ignored` is just so much easier to digest than `closed` (or alternative).